pkg/framework/didresolver/api.go

```diff
@@ -72,6 +72,6 @@ type Opt func(opts *didResolverOpts)
 // WithDidMethod to add did method
```

Comment:
Should update the package docs to indicate that regular expressions are supported and DID methods are checked in the order added.

Reply:
I add comment

Reply:
done